hangman.py

```diff
@@ -0,0 +1,311 @@
+# but you will have to know how to use the functions
+# (so be sure to read the docstrings!)
+import random
+import string
+
+WORDLIST_FILENAME = "words.txt"
+
+secret_word = str()
+letter_guessed = str()
+def load_words():
+    """
+    Returns a list of valid words. Words are strings of lowercase letters.
+    
+    Depending on the size of the word list, this function may
+    take a while to finish.
+    """
+    print("Loading word list from file...")
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    # line: string
+    line = inFile.readline()
+    # wordlist: list of strings
+    wordlist = line.split()
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+    
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+# end of helper code
+
+# -----------------------------------
+
+# Load the list of words into the variable wordlist
+# so that it can be accessed from anywhere in the program
+wordlist = load_words()
+
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: list (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    for i in range(len(secret_word)):
+        if secret_word[i] not in letters_guessed:
+            return False
+    return True
+   #letters_guessed = input("enter letter:").strip(' ')
+   #letters_guessed.lower()
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+
+
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    splited_word = list(secret_word)
+    for i in range(len(secret_word)):
+        if secret_word[i] not in letters_guessed:
+            splited_word[i] = "_ "
+    return  "".join(splited_word)
+
+
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    avaible_letters = "abcdefghijklmnopqrstuvwxyz"
+    avaible_letters = set(avaible_letters)
+    letters_guessed = set(letters_guessed)
+    avaible_letters = list(avaible_letters - letters_guessed)
+    avaible_letters.sort()
+    return str(avaible_letters).replace(", ", "").replace("'","")[1:-1]
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+
+
+
+
+def hangman(secret_word):
```

Comment:
This implementation has different output from what requested by task. Please - fix it